source/route_auth.js

```diff
@@ -12,27 +12,27 @@ angular.module( "vokal.RouteAuth", [ "LocalStorageModule" ] )
         redirectPath = path;
     };
 
-    this.$get = [ "$q", "$location", "$rootScope", "localStorageService",
+    this.$get = [ "$q", "$location", "$rootScope",
 
-        function ( $q, $location, $rootScope, localStorageService )
+        function ( $q, $location, $rootScope )
         {
             var service = {
-
+                storageType: localStorage,
```

Comment:
I don't think I agree with attaching what should be `window.localStorage` to the service.

I'm guessing this is how you'd manage to switch storage between `localStorage` and `sessionStorage` while the app is running. I think this could be problematic without a wrapping function to transition or clear the existing data from the previous storage. Possibly use another variable inside the service function to reference the active storage, and pass a method back to set the storage type instead?